examples/integrated_gradients_transformers.ipynb

```diff
@@ -11,9 +11,9 @@
    "cell_type": "markdown",
```

Comment:
<p>A few more comments:</p><p> </p><ul><li>"We go with the embedding layer" --&gt; something more explicit like "We calculate attributions to the outputs of the embedding layer..."</li><li>"By considering a hidden layer..." --&gt; maybe something like "If we considered a hidden layer instead, we would inevitably capture..." to make the difference between embedding and hidden layer explicit</li><li>"one-to-one mapping" --&gt; be explicit what the mapping considered is, e.g. "one-to-one mapping between layer outputs and input tokens."</li><li>"can not" --&gt; "cannot"</li><li>"Note that you can extract any other layer." --&gt; be more concrete, e.g. "That being said, you can use any other layer and compute attributions to the outputs of it instead."</li><li>"Uncomment the preferred one." --&gt; delete</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/alibi/pull/537/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>